src/prefect/client/client.py

```diff
@@ -1308,6 +1308,28 @@ def set_task_run_name(self, task_run_id: str, name: str) -> bool:
 
         return result.data.set_task_run_name.success
 
+    def cancel_flow_run(self, flow_run_id: str) -> bool:
+        """
+        Cancel the flow run by id
+
+        Args:
+            - flow_run_id (str): the id of the flow run
+
+        Returns:
+            - bool: whether or not the flow run was canceled
+        """
+        mutation = {
+            "mutation($input: cancel_flow_run_id_input!)": {
+                "cancel_flow_run(input: $input)": {
+                    "success": True,
+                }
+            }
+        }
+        result = self.graphql(
+            mutation, variables=dict(input=dict(flow_run_id=flow_run_id))
```

Comment:
Can someone point me to where the work of this mutation is actually done? I took a quick glance at `server` but didn't see it.

Reply:
https://github.com/PrefectHQ/server/blob/9b315589fde7ca08ef7d1c49c23792bc0d2c4dfd/src/prefect_server/api/states.py#L210

Reply:
Great thanks, and how does this get mapped to the `states` namespace?

Reply:
https://github.com/PrefectHQ/server/blob/7380c7f97d13784dfc5db4dc044be340fe0a77bd/src/prefect_server/graphql/states.py#L96